### DISCUSSION TITLE
Regain focus from editorjs

### DISCUSSION THREAD

**Discussion by antonbashmakov** [codex-team/editor.js#2653] [Q&A]
When focused the editor intercepts with key events like arrow down and arrow up. Is it possible to regain control of the event handling when last or first block been reached? 

So we have this page with an input field and the editor underneath. User clicked on the editors first block and pressed arrow up on the keyboard. I need to focus on the input field. Is it possible? 

<img width="880" alt="Screenshot 2024-03-11 at 14 35 03" src="https://github.com/codex-team/editor.js/assets/7541650/6abd7ed8-84f1-44d2-a75d-88471386621a">

Thanks

**Comment by neSpecc** [comment]
You can use `Tab` or `Shift+Tab` to navigate from editor to another form inputs. It is a default browser behaviour and it was supported in 2.29. 

Using arrows for navigation between inputs is not a native behaviour — most html forms does not work this way. Arrows used for Caret moving inside an input
